### ISSUE TITLE
Errors with the differential abundance analysis results

### ISSUE THREAD

**Issue by BinhongLiu** [YuLab-SMU/MicrobiotaProcess#12]
Dear Yu,
       Thanks for the development of this wonderful package, which can help me a lot with the microbiome data analysis. It came an error showed that "Error in get_call(obj, "classgroup") : 缺少参数"argres",也没有缺省值" when showed the differ result after the diff_analysis analysis. Also, I can not plot the differ result analyzed from the demo data kostic2012crc. The error is still there after I update the package to the development version from github. Could you help me with this error, please?
Thank you!
Hongbin Liu

**Comment by xiangpin** [comment]
Because you don't provide code, I don't know the details. But I guess you used old parameter `class`. The parameter had been replaced `classgroup`.

**Comment by BinhongLiu** [comment]
Yeah, you were right! I just tried the test script here https://mp.weixin.qq.com/s/iHcQpKs5Jo28Q_FiAzdTuw. The parameter should be classgroup, but not class. Thanks!

**Comment by xiangpin** [comment]
Please refer the `Document` in https://bioconductor.org/packages/release/bioc/html/MicrobiotaProcess.html
